chat-sdk-ui/src/main/AndroidManifest.xml

```diff
@@ -97,6 +97,9 @@
             android:windowSoftInputMode="stateHidden"
             android:theme="@style/ChatSDKSlidingTransactionTheme"/>
 
+        <activity android:name=".SendMultipleImagesAtOnceActivity"
```

Comment:
Can you change the name of this activity to `ImageGalleryActivity`

Reply:
FYI, to change a name right click and chose "Refactor" then "Rename". This will rename all the instances of the class automatically.